pkg/api/admin_users.go

```diff
@@ -33,6 +35,10 @@ func AdminCreateUser(c *models.ReqContext, form dtos.AdminCreateUserForm) Respon
 			return Error(400, models.ErrOrgNotFound.Error(), nil)
 		}
 
+		if err == models.ErrUserAlreadyExists {
+			return Error(412, fmt.Sprintf("User with email %s and/or login %s already exists", form.Email, form.Login), err)
```

Comment:
We call the login param username in the form. I also changed `and/or` to just `or`, since there's nothing that implies that the or is exclusive and it's a slightly more natural sentence (:joy: not sure any sentence with an email in the middle is natural).

And consider adding quotes around the email and username if the user uses whitespace in them to make the error clearer (not sure we allow that, but in case)

```suggestion
			return Error(412, fmt.Sprintf("User with email '%s' or username '%s' already exists", form.Email, form.Login), err)
```

Reply:
Thanks for the suggestion! Indeed, your sentence really looks more natural 👌